page-data.php

```diff
@@ -30,30 +30,45 @@
 </main></div>
 
 <aside id="secondary">
+  <div id="js-TJIChartViewFilters"></div>
 </aside>
 
 <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.10/lodash.min.js"></script>
```

Comment:
Should we maybe add a comment here that, if you add any scripts needed by TJIChartView, to add them to the 'Dependencies' comment there? I just worry about this getting out of sync.

Reply:
Can do.